source/instructors/setupcourses/assignment-settings/prerequisite-assignments.rst

```diff
@@ -34,6 +34,6 @@ The **Copy Workspace From** setting allows you to copy content from one assignme
 
 When the assignment is created by the student, the workspace files/folders (excluding **.guides**, **.git** folder, any node modules and any **.codio**, **.codio-menu** or **.settings** files) of the assignment selected in the dropdown will be copied into the prerequisite assignment's workspace and will be included in the new assignment. 
 
-If a file of the same name exists in both assignnments, the most recent changes to the initial assignment and will be updated in the subsequent assignment. This will apply to changes that students made.
+If a file of the same name exists in both assignnments, the most recent changes to the initial assignment will be updated in the subsequent assignment. This will apply to changes that students made.
```

Comment:
This is still a little confusing - is the file from the prerequisite assignment merged into the current file or does it over-write it? I do not know how this is supposed to work.

Reading the YouTrack, it sounds like the files from the prerequisite assignment are copied over to the new assignment in complete at the time that the new assignment is opened.

Reply:
1: "into" the prerequisite assignment not "from" as per last comment
2: reads to me that if there is file of same name and in the previous assignment there was a change then that is what is then copied into the pre requisite assignments file as the pre requisite assignment is first opened by the student.  copy or overwritten reads same to me